File: workflows/pythonSQL2in2out/code/src/main/python/graph/Reformat1.py
from pyspark.sql import SparkSession
from pyspark.sql.types import *
from pyspark.sql.functions import *
from ..config import Config
from ..prophecylibs import *

@Visual(id = "Reformat1", label = "Reformat1", x = 530, y = 229, phase = 0)
def Reformat1(spark: SparkSession, _in: DataFrame) -> Reformat:
    out = _in.select()

    return out

File: workflows/detailed_stats_experiments/code/src/main/scala/graph/Reformat1.scala
package graph

import org.apache.spark.sql.types._
import io.prophecy.libs._
import io.prophecy.libs.UDFUtils._
import io.prophecy.libs.Component._
import io.prophecy.libs.DataHelpers._
import io.prophecy.libs.SparkFunctions._
import io.prophecy.libs.FixedFileFormatImplicits._
import org.apache.spark.sql.ProphecyDataFrame._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import config.ConfigStore._
import graph._

@Visual(id = "Reformat1", label = "Reformat1", x = 792, y = 258, phase = 0, detailedStats = true)
object Reformat1 {

  def apply(spark: SparkSession, in: DataFrame): Reformat = {
    import spark.implicits._

    val out = in.select(
      col("customer_id"),
      col("last_name"),
      col("country_code")
    )

    out

  }

}
